src/App.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import Auth from './containers/AuthForm/Auth';
+import { Redirect, Route, Switch } from 'react-router';
+import { NavLink } from 'react-router-dom';
+import Button from './components/UI/Button/Button';
+import { Dispatch } from 'redux';
+import { logOut } from './store/auth/actions';
+import { connect } from 'react-redux';
+import { RootState } from './store';
+import { Fragment } from 'react';
+
+interface StateProps {
+  isAuthenticated: boolean
+}
+
+interface DispatchProps {
+  onLogOut: () => void
+}
+
+type Props = StateProps & DispatchProps
+
+class App extends Component<Props> {
+
+  logOutHandler = () => {
+    this.props.onLogOut();
+  };
+
+  render() {
+    // ToDo: Create a navigation component
+
+    const content = () => {
+      if (this.props.isAuthenticated) {
+        return (
+          <Fragment>
+            {/*ToDo: rid off from the inline-style */}
```

Comment:
Hint: take a look at `styled-components` or some ready-made UI kits — I highly recommend http://material-ui.com

Reply:
Thank you for your advice!
I'll use styled-components from material UI in the next PR